infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/environment/util/EntryPoint.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.workspace.infrastructure.kubernetes.environment.util;
+
+import java.util.List;
+import java.util.Objects;
+
+/** Represents an entry-point definition parsed from a string using the {@link EntryPointParser}. */
+public final class EntryPoint {
+
+  private final List<String> command;
+  private final List<String> arguments;
+
+  public EntryPoint(List<String> command, List<String> arguments) {
+    this.command = command;
+    this.arguments = arguments;
+  }
+
+  public List<String> getCommand() {
+    return command;
```

Comment:
It will be safer if this POJO has getters with null checking, as we have in other POJOs https://github.com/eclipse/che/blob/5907d1cad5c621e7d726cbeb27c89127fcc4d28b/wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/model/impl/WorkspaceConfigImpl.java#L172-L178